src/Common/tests/System/Net/Capability.Sockets.cs

```diff
@@ -47,5 +47,10 @@ public static bool IPv4Support()
         {
             return Socket.OSSupportsIPv4;
         }
+
+        public static bool UnixDomainSocketsSupport()
+        {
+            return !RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
```

Comment:
The purpose of the Capabilities class is to detect run-time support: e.g. while running on a Windows machine, tests exercising IPv4 will fail if the protocol isn't installed on the machine. 

I believe that the correct check would be similar to the OSSupportsIPv4 implementation.